opentelekomcloud/services/antiddos/resource_opentelekomcloud_antiddos_v1.go

```diff
@@ -25,6 +25,8 @@ func ResourceAntiDdosV1() *schema.Resource {
 		UpdateContext: resourceAntiDdosV1Update,
 		DeleteContext: resourceAntiDdosV1Delete,
 
+		DeprecationMessage: "AntiDDoS protection for elasticIP is provided by default and shouldn't be created manually.",
```

Comment:
Should be "elastic IP", not concatinated string

Reply:
done